traceview/__init__.py

```diff
@@ -207,7 +207,9 @@ def controllers(self):
         return self._controllers.get()
 
     def delete(self, host_id, *args, **kwargs):
-        """ Delete an existing host.
+        """ **Deprecated:** please use delete_host or delete_app.
```

Comment:
@dankosaur two small things:

1. Uppercase `DEPRECATED`
2. Since this was *only* for deleting a host, I think we should simply say: "Please use `delete_host` instead."